core/Dockerfile

```diff
@@ -1,3 +1,9 @@
+FROM node:current-buster-slim AS frontendBuilder
+
+# Build frontend
+COPY frontend /home/pi/frontend
+RUN cd /home/pi/frontend && yarn install && yarn build
```

Comment:
>This is wrong, as you can see in the line 33 of the current file, the multistage will take care of the frontend build process.

Could you elaborate on that? Line 33 is merely copying the dist folder *after* its build (on line 5).

> The copy of the frontend is also not correct, since this project is missing the dockerignore, otherwise we are going break the docker cache system calculation that is based in file fingerprint.

You've already stated the missing dockerignore...